client/src/App.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
Looks to me that changes for `body` should be in https://github.com/ghilannicoleta/la-fontana-app/blob/main/client/src/index.css#L5.
And `App.css` should be removed from the project, as this project uses tailwind